apps/studio.giselles.ai/components/task/task-header.tsx

```diff
@@ -90,6 +94,17 @@ export function TaskHeader({
 			<div className="mx-auto pt-2">
 				{/* App Summary Section */}
 				<div>
+					{isBackLinkVisible ? (
+						<div className="mb-2">
+							<Link
+								href={backHref}
+								className="inline-flex items-center gap-2 text-[13px] text-text-muted/70 hover:text-text-muted transition-colors"
+							>
+								<ArrowLeft className="h-4 w-4" aria-hidden />
+								Back to Tasks
+							</Link>
+						</div>
+					) : null}
```

Comment:
The layout height changes depending on whether isBackLinkVisible is enabled or disabled, so when you run it in the Playground, the screen will flicker during and after Loading.